### ISSUE TITLE
DAC 16 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 03

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#16]
<img width="364" alt="DAC_Colour_Contrast_Non_Text_03" src="https://user-images.githubusercontent.com/75850106/155558497-939ae59c-3c23-4e44-99b0-33c5b40cd022.png">

### Issue
The map that shows deprived areas has a lot of different colours. The majority of the colours used are not greater than a 3:1 contrast ratio against adjacent colours. This means that low vision users may have difficulty seeing the map information. A bar graph is also present, and the bar colours used are not greater than 3:1 compared to their adjacent colours.

_Red on Red_
Foreground: #D6604D Background: #B2182B
The contrast ratio is: 1.8:1 1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

_Blue on Blue_
Foreground: #D1E5F0 Background: #92C5DE
The contrast ratio is: 1.4:1 1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

_Blue map colour_
Foreground: #4393C3 Background: #EDEEEA
The contrast ratio is: 2.9:1 1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

_Salmon map colour_ 
Foreground: #FDDBC7 Background: #F2F3F0
The contrast ratio is: 1.2:1 1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

<img width="599" alt="DAC 16 Current code ref" src="https://user-images.githubusercontent.com/75850106/155558763-a1b27fad-18c4-45c1-87cf-41e7dfcf0880.png">

### Suggested Solution
Non-text colours must have a colour contrast of 3:1 or greater against adjacent colours. Check that all of the colours used for the map have a minimum 3:1 colour contrast.
